src/wallet/rpc/spend.cpp

```diff
@@ -1644,7 +1643,6 @@ RPCHelpMan walletcreatefundedpsbt()
     bool rbf{wallet.m_signal_rbf};
     const UniValue &replaceable_arg = options["replaceable"];
     if (!replaceable_arg.isNull()) {
-        RPCTypeCheckArgument(replaceable_arg, UniValue::VBOOL);
         rbf = replaceable_arg.isTrue();
```

Comment:
This is a behavior change. Previously it would throw on non-bool, now it will translate it to false

Reply:
Sounds like a bug in our UniValue fork that will affect master as well?